external-crates/move/crates/move-compiler/src/naming/translate.rs

```diff
@@ -694,6 +817,13 @@ fn explicit_use_fun(
                 .add_diag(diag!(Declarations::InvalidUseFun, (loc, msg)));
             None
         }
+        ResolvedFunction::Var(_) => {
+            let msg = "Invalid 'use fun'. Cannot use a local variable as a method";
```

Comment:
```suggestion
            let msg = "Invalid 'use fun'. Local variables are not functions";
```

(maybe?)

Reply:
What about lambdas? I know they aren't exactly functions but... they might feel like it to a user

Reply:
EDIT: Ah! I forgot, I just excluded this case explicitly now... which might be more confusing

Reply:
Yeah, I think now this should be marked as unreachable?